smart_open/smart_open_lib.py

```diff
@@ -754,11 +754,11 @@ def HttpOpenRead(parsed_uri, mode='r', **kwargs):
 
     response = HttpReadStream(url, **kwargs)
 
-    fname = url.split('/')[-1]
+    fname = urlsplit(url, allow_fragments=False).path.split('/')[-1]
```

Comment:
Sorry, would need an example. In what I tried path is the URL without the parameters.